src/main/msp/msp.c

```diff
@@ -1147,7 +1147,7 @@ static bool mspProcessOutCommand(mspDescriptor_t srcDesc, int16_t cmdMSP, sbuf_t
 
             for (int i = 0; i < 3; i++) {
 #if defined(USE_ACC)
-                sbufWriteU16(dst, lrintf(acc.accADC[i] / scale));
```

Comment:
`float scale;` above; and use '1.0f/8','1.0f/4', . And eventually, use `256.0f * acc.dev.acc_1G_rec`)

Reply:
I don't understand the second part of your comment. Where should I use `256.0f * acc.dev.acc_1G_rec`? I refactored the necessary parts and wanted to make as little changes as possible to the existing codebase in this PR. But I guess small optimizations won't hurt.

Reply:
This is converting accADC to 1/256 g units. It did the job for when accADC was `int16_t`. 
With float, we can use reciprocal value of acc_1G (acc_1G_rec) and get the result with single multiplication.
But it is unrelated to this PR.

Reply:
Actually, no scaling is IMO performed in master now.